index.html

```diff
@@ -1286,9 +1286,7 @@ <h2>DID Subject</h2>
 However, the fully resolved <a>DID document</a> always contains a valid
 <code><a>id</a></code> property. The value of <code><a>id</a></code> in the
 resolved <a>DID document</a> MUST match the <a>DID</a> that was
-resolved, or be populated with the equivalent canonical <a>DID</a>
-specified by the <a>DID method</a>, which SHOULD be used by the resolving
-party going forward.
+resolved.
```

Comment:
I'm in favor of changing this back, I always felt uncomfortable about allowing anything other than the original DID in the `id` property value.